automation/check-patch.e2e-k8s.sh

```diff
@@ -9,7 +9,6 @@
 
 teardown() {
     make cluster-down
-    cp $(find . -name "*junit*.xml") $ARTIFACTS
```

Comment:
Why are we dropping this?

Reply:
With the kubevirt.io junit reporter we can specify the path directly [1]  and there we just use $ARTIFACTS so no need for cp

[1] https://github.com/nmstate/kubernetes-nmstate/pull/555/files#diff-2da872edd4c334f4c06df2de5bf629c7R32